### PR TITLE
PEO-412 Wrap or convert thor commands into a rake tasks

### PR DESCRIPTION
**Pull request by D3N**
[PEO-412](https://garage.atlassian.net/browse/PEO-412)

### Description

Wrap or convert thor commands into a rake tasks

### Before submitting the merge request make sure the following are checked

- [x] Followed the style guidelines of this project
- [x] Performed a self-review of own code
- [x] Wrote the tests that prove that fix is effective/that feature works

### CODE REVIEW

**Review by rostikkkk2 — DISMISSED** [review]

File: lib/inquisition/tasks/inquisition.rake [hunk 1]
```diff
@@ -0,0 +1,4 @@
+desc 'Run Inquisition'
```
Comment (chubchenko) [inline lib/inquisition/tasks/inquisition.rake]: I am sorry, it's my fault, but now I think would be better to create a `run` task wrapped by `inquisition` namespace

File: spec/inquisition/tasks/inquisition_rake_spec.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+require 'rake'
```
Comment (chubchenko) [inline spec/inquisition/tasks/inquisition_rake_spec.rb]: Propose to create a `shared_context` with the next functionality:
- require rake task
- task -> returns the task
- define_task

Comment (chubchenko) [reply spec/inquisition/tasks/inquisition_rake_spec.rb]: Propose to rename this file to `inquisition_spec.rb`

File: spec/inquisition/tasks/inquisition_rake_spec.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    Rake.application.rake_require 'inquisition/tasks/inquisition'
+    Rake::Task.define_task(:environment)
+    allow(Inquisition::Collector).to receive(:new) { collector }
+    allow(collector).to receive(:call)
+  end
+
+  it do
+    Rake.application.invoke_task 'inquisition'
+    expect(Inquisition::Collector).to have_received(:new).exactly(1).times
+    expect(collector).to have_received(:call).exactly(1).times
```
Comment (chubchenko) [inline spec/inquisition/tasks/inquisition_rake_spec.rb]: `expect(collector).to have_received(:call).once`

**Review by chubchenko — CHANGES_REQUESTED** [review]

**Review by chubchenko — APPROVED** [review]

**Pull request merged**